docs/README.md

```diff
@@ -65,43 +51,20 @@ The Shelf is a theme block responsible for showing a **collection of products**
       "flex-layout.row#main",
       "shelf.relatedProducts"
     ]
-  },
+  }
+}
 ```
 
-### Shelf 
+| Prop name              | Type                                   | Description                                                                                                                                                                                                                               | Default value |
+| ---------------------- | -------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `category`             | `String`                               | Category ID of the listed items. For sub-categories, use "/" (e.g. "1/2/3")                                                                                                                                                               | -             |
+| `specificationFilters` | `Array({ id: String, value: String })` | Specification Filters of the listed items.                                                                                                                                                                                                | []            |
+| `collection`           | `String`                               | Filter by collection.                                                                                                                                                                                                                     | -             |
```

Comment:
| `collection`           | `String`                | Collection ID of the Shelf listed items.                                                                                                                                                                                                                  | `undefined`          |